src/config/shared.js

```diff
@@ -32,3 +32,38 @@ export function extractBucketMetadata(transformedData) {
   etc.bucketSortOrder = sorter;
   return etc;
 }
+
+export const facets = {
```

Comment:
Let's wait to do this for another PR. The default value for channels in other products are (for instance) not nightly. They're usually release. It'll be clearer what belongs in shared when we get our bearings, which will probably happen once people can see the data in GLAM.